<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormLeft</class>
 <widget class="QWidget" name="FormLeft">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>754</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0,0,0,0,0,0,0,0">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="layout0" stretch="0">
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="SliderBar" name="slider_bar" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout1">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_image">
       <property name="minimumSize">
        <size>
         <width>15</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>15</width>
         <height>30</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="background_color" stdset="0">
        <string>1</string>
       </property>
       <property name="CustomStyle" stdset="0">
        <string>switch</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttongroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_image">
       <property name="maximumSize">
        <size>
         <width>235</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="text">
        <string>Image  Operating   </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="border_no" stdset="0">
        <bool>true</bool>
       </property>
       <property name="background_color" stdset="0">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_image" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="background_color" stdset="0">
        <string>3</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="layout_image">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_image1">
            <property name="text">
             <string>载入</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_image2">
            <property name="text">
             <string>导出</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_image3">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>撤销</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_image4">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>重做</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="layout_imageshow">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <item row="0" column="2">
           <widget class="QPushButton" name="btn_menuimage2">
            <property name="text">
             <string>图像增强</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="btn_menuimage1">
            <property name="text">
             <string>灰度变换</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="btn_menuimage0">
            <property name="text">
             <string>几何变换</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="btn_menuimage5">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>开发测试</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="btn_menuimage4">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>颜色模型形</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="btn_menuimage3">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>形态学处理</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_mode">
       <property name="minimumSize">
        <size>
         <width>15</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>15</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="background_color" stdset="0">
        <string>1</string>
       </property>
       <property name="CustomStyle" stdset="0">
        <string>switch</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttongroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_mode">
       <property name="maximumSize">
        <size>
         <width>235</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Model  Operating   </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="border_no" stdset="0">
        <bool>true</bool>
       </property>
       <property name="background_color" stdset="0">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout4">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_mode" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="background_color" stdset="0">
        <string>3</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0,0,0">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_18">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btn_mode1">
            <property name="text">
             <string>分割</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_mode2">
            <property name="text">
             <string>剪裁</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_mode3">
            <property name="text">
             <string>修剪</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_mode4">
            <property name="text">
             <string>平滑</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_mode5">
            <property name="text">
             <string>细分</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_mode6">
            <property name="text">
             <string>细化</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_20">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btn_mode11">
            <property name="text">
             <string>细分(CGAL)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_mode12">
            <property name="text">
             <string>自相交检测</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_mode13">
            <property name="text">
             <string>封闭</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_21">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btn_mode20">
            <property name="text">
             <string>三维重构</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_mode21">
            <property name="text">
             <string>载入</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_mode22">
            <property name="text">
             <string>导出</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_mode23">
            <property name="text">
             <string>撤销</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout5">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_dcm">
       <property name="minimumSize">
        <size>
         <width>15</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>15</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="background_color" stdset="0">
        <string>1</string>
       </property>
       <property name="CustomStyle" stdset="0">
        <string>switch</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttongroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_dcm">
       <property name="maximumSize">
        <size>
         <width>235</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Dcm  Operating   </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="border_no" stdset="0">
        <bool>true</bool>
       </property>
       <property name="background_color" stdset="0">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout6">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_dcm" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout7">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_images">
       <property name="minimumSize">
        <size>
         <width>15</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>15</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="background_color" stdset="0">
        <string>1</string>
       </property>
       <property name="CustomStyle" stdset="0">
        <string>switch</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttongroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_images">
       <property name="maximumSize">
        <size>
         <width>235</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Images  Operating   </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="border_no" stdset="0">
        <bool>true</bool>
       </property>
       <property name="background_color" stdset="0">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout8">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_images" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btn_images1">
            <property name="text">
             <string>左侧图片读取</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_images2">
            <property name="text">
             <string>右侧图片读取</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layout9">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QTextEdit" name="textedit_information">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>100</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu'; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CpuMemoryLabel" name="label_cpu">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="border_no" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CpuMemoryLabel</class>
   <extends>QLabel</extends>
   <header>cpumemorylabel.h</header>
  </customwidget>
  <customwidget>
   <class>SliderBar</class>
   <extends>QWidget</extends>
   <header>sliderbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttongroup"/>
 </buttongroups>
</ui>
